documentation/api-scenario/how-to/IntegrateWithTestProxy.md

```diff
@@ -0,0 +1,87 @@
+# How to integrate test-proxy with API Scenario test
+
+In this section, we'll show you how to make test-proxy recording with API Scenario test, assuming you've already learnt how to use API Scenario test. If not, please go to [QuickStart](./QuickStart.md).
+
+Test-proxy is a tool that provides out-of-process record/playback capabilities compatible with any language. It also supports pushing/restoring the API test recordings from external git repository. See [how to install test-proxy](https://github.com/Azure/azure-sdk-tools/blob/main/tools/test-proxy/Azure.Sdk.Tools.TestProxy/README.md#installation)
+
+## Prepare assets.json file
+
+The assets.json file is a configuration file used by test-proxy to push/restore recording to/from external git repository. Create an assets.json file under `scenarios/` folder with following content:
+
+```json
+{
+  "AssetsRepo": "Azure/azure-sdk-assets",
+  "AssetsRepoPrefixPath": "",
+  "TagPrefix": "apitest/<ServiceName>/<package>"
+}
+```
+
+Take appConfiguration as an example,
+
+```
+specification/appconfiguration/data-plane
+├── Microsoft.AppConfiguration
+│   └── stable
+│       └── 1.0
+│           ├── appconfiguration.json
+│           ├── examples
+│           │   ├── CheckKeyValue.json
+│           │   ├── PutLock.json
+│           │   └── PutLock_IfMatch.json
+│           └── scenarios
+│               └── assets.json         <----- check-in assets.json here
+└── readme.md
+```
+
+## Make recording
+
+1. Start test-proxy under the repository root folder
+
+```bash
+test-proxy start
+```
+
+By default, it listens to 5000/5001 ports for http/https respectively.
+
+2. Run API Scenario test
+
+```bash
+oav run <your-scenario-file>.yaml -e <your-env>.json --testProxy http://localhost:5000 --testProxyAssets <your-assets-file>.json
+```
```

Comment:
The option `--testProxyAssets` is supported from 3.3.0 version. Install latest or beta version oav: `npm i -g oav@beta` or `npm i -g oav`.